### ISSUE TITLE
Bypass disabled by script for any farm by placing a beacon

### ISSUE THREAD

**Issue by boldviking** [pyanodon/pyalienlife#227]
So I observed the following behavior on today's stream.

Place down a mk2 sap farm without any modules.
Observe disabled by script status.
Place down diet beacon covering the mk2 sap farm.
Mk2 sap farm will start working as its "disabled by script" status gets removed and without any modules inside the building itself.

This seems to works for any farm. I tried it for a kicalk farm with the same result.

So in short, pacing down a beacon will clear the disabled by script status for any farm.

**Comment by kingarthur91** [comment]
fixed it. thanks for letting us know
